tests/micropython/test_cases/test_animation.py

```diff
@@ -0,0 +1,54 @@
+#! ../../../../../../../lv_micropython/ports/unix/build-standard/micropython ../run_testcase.py
+#LVGL MicroPython binding tester script: Animation to change widget-position, start & ready callbacks
+
+LV_TESTANIM_FRAMERATE = 10 #ms
+LV_TESTANIM_LOOPCOUNT = 30
+
+
+#test-objects:
+
+lv_testbutton = lv.button( lv.scr_act() )
```

Comment:
The `lv_` prefix is not required for variables like these.  Having the `lv_` prefix suggests that it is related to LVGL and not the user's code.

Feel free to call it just `testbtn`.